<ui version="4.0" >
 <class>ProjectSelectionPage</class>
 <widget class="QWidget" name="ProjectSelectionPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>475</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTreeView" name="templateView" />
   </item>
   <item>
    <widget class="QGroupBox" name="propertiesBox" >
     <property name="title" >
      <string>Properties</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="KUrlRequester" name="locationUrl" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="finalLocationLabel" >
        <property name="text" >
         <string>Final Location:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="locationLabel" >
        <property name="text" >
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="appNameLabel" >
        <property name="text" >
         <string>Application Name:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="finalLocationLabel" >
        <property name="text" >
         <string>(invalid)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="appNameEdit" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QLineEdit</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
